src/web/components/SharingPermission/bulkAddPermissionsHelpers.ts

```diff
@@ -0,0 +1,57 @@
+import { formatStringsWithSeparator, getArticle } from '../../utils/textHelpers';
+
+export const getDefaultPublisherCheckboxState = (participantTypeNames: string[]) => {
+  if (participantTypeNames?.includes('Data Provider')) return true;
```

Comment:
I found types are used a lot in this file, could we change them to constant or enum? Just in case we rename or add a new participant type.

NIT:not sure if something like 
```
const types = ['Data Prodiver', 'DSP] 
participantTypesNames.some(type =>types.includes(type))
```
make sense to you, I think then this could be reused rather than multiple 
`||participantTypeNames?.includes()`

Reply:
✔ good suggestion! I was struggling to think of a way to make this code better, so that definitely helps.